charypar/README.md

```diff
@@ -1,6 +1,38 @@
 # Viktor Charypar
 
-I'm a Tech Director in Cell A. At some point I'll write more about what I do and tech I like. Maybe.
+[@charypar](https://twitter.com/charypar)
+
+> Tech Director in Cell A, puzzles and aviation nerd, [tapir](https://www.bioexpedition.com/tapir/) and pig enthusiast.
+
+I'm on a mission to simplify DevOps workflows and remove unnecessary complexity, which we've collectively piled on  
+over the years, and let us focus on creating great software experiences for the humans who use our software. I don't
```

Comment:
looks like these new-lines render as line breaks, which is a bit weird. You might want to join into one long line and use soft wrap